src/Stan/Inspection/Infinite.hs

```diff
@@ -22,29 +22,24 @@ module Stan.Inspection.Infinite
     , stan0103Meta
 
       -- * All inspections
-    , infiniteInspections
-    , infiniteInspectionsIds
+    , infiniteInspectionsMap
     ) where
 
 import Stan.Category (infinite)
 import Stan.Core.Id (Id (..))
-import Stan.Inspection (Inspection (..))
+import Stan.Inspection (Inspection (..), InspectionsMap)
 import Stan.NameMeta (NameMeta (..), mkBaseListMeta)
 import Stan.Severity (Severity (..))
 
 
--- | All infinite 'Inspection's.
-infiniteInspections :: [Inspection]
-infiniteInspections =
+-- | All infinite 'Inspection's map from 'Id's.
+infiniteInspectionsMap :: InspectionsMap
+infiniteInspectionsMap = fromList $ map (\i -> (inspectionId i, i))
```

Comment:
Looks like `\i -> (inspectionId i, i)` can be replaced by [`mapToFst`](http://hackage.haskell.org/package/relude-0.6.0.0/docs/Relude-Extra-Tuple.html#v:mapToFst). But we have an open issue in `relude` regarding better naming for this function...